i18n4go/cmds/rewrite_package.go

```diff
@@ -148,21 +153,50 @@ func (rp *rewritePackage) Printf(msg string, a ...interface{}) (int, error) {
 }
 
 func (rp *rewritePackage) Run() error {
-	var err error
 
 	if rp.options.FilenameFlag != "" {
-		if err = rp.loadStringsToBeTranslated(rp.I18nStringsFilename); err != nil {
+		if err := rp.loadStringsToBeTranslated(rp.I18nStringsFilename); err != nil {
 			return err
 		}
-		err = rp.processFilename(rp.options.FilenameFlag)
+		if err := rp.processFilename(rp.options.FilenameFlag); err != nil {
+			return err
+		}
+		rp.I18nStringsFilePaths = []string{rp.I18nStringsFilename}
 	} else {
-		err = rp.processDir(rp.options.DirnameFlag, rp.options.RecurseFlag)
+		if err := rp.processDir(rp.options.DirnameFlag, rp.options.RecurseFlag); err != nil {
+			return err
+		}
+	}
+
+	if err := rp.generateBindataFromI18nStrings(); err != nil {
+		return err
 	}
 
 	rp.Println()
 	rp.Println("Total files parsed:", rp.TotalFiles)
 	rp.Println("Total rewritten strings:", rp.TotalStrings)
-	return err
+	return nil
+}
+
+func (rp *rewritePackage) generateBindataFromI18nStrings() error {
+	if rp.RootPackageName != "" {
+		rp.Println("i18n4go: adding binary go file to package:", rp.RootPackageName, " to output dir:", rp.OutputDirname)
```

Comment:
Can we do this w/o introducing new strings since we now have to track this and i18n it?

Reply:
Sure we don't really need the print statement here so I will just remove it.